aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,26 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## June 4, 2022 - July 8, 2022
+
+### Stay Current with the Latest Changes
+
+- **Tabs**
+  - We've enhanced responsive behavior and updated the guidance.
```

Comment:
can we add a link to the example on the site 
https://design-system.hpe.design/components/tabs#responsive-behavior

Reply:
done